config/database.yml.sample

```
@@ -1,12 +1,27 @@
 ---
-development: &local
-  adapter: trilogy
-  host: 127.0.0.1
-  port: 3306
-  database: lobsters_dev
-  username: root
-  password: localdev
+development:
+  primary: &local
+    adapter: trilogy
+    host: 127.0.0.1
+    port: 3306
+    database: lobsters_dev
+    username: root
+    password: localdev
+  cache:
+    database: storage/development_cache.sqlite3
```

```suggestion
    database: tmp/cache/cache.sqlite3
```

You can have your PR rm `storage/`, it's the file store we're replacing.